CMakeLists.txt

```diff
@@ -507,6 +507,11 @@ ELSE ()
 	ENDIF ()
 ENDIF()
 
+CHECK_FUNCTION_EXISTS(regcomp_l HAVE_REGCOMP_L)
```

Comment:
Yes, that's what it looks like from its documentation. I wonder if `CHECK_FUNCTION_EXISTS` might work here as well, since it's system-provided.